src/main/java/uk/gov/ons/census/casesvc/service/UacService.java

```diff
@@ -152,4 +156,28 @@ public UacQidLink findByQid(String questionnaireId) {
 
     return uacQidLinkOpt.get();
   }
+
+  private String mapQuestionnaireTypeToFormType(String qid) {
```

Comment:
Not sure what our policy is on testing mappings like this? The current unit tests only check that questionnaireType 1  maps correctly

Reply:
I thought about doing an exhaustive unit test and then I thought "I can't be bothered to test 15 lines of switch statement which is just about the simplest possible code I've ever written", especially as a private method. We can have exhaustive AT scenarios if people are going to go mad with testing every combination and permutation imaginable, but does the unit test give us some degree of regression protection? Yes.